Sources/SwiftSyntaxBuilder/Buildables.swift.gyb

```diff
@@ -229,15 +251,37 @@ public struct ${node.syntax_kind}: SyntaxBuildable {
 
 %   end
 %   if node.is_buildable() or node.is_syntax_collection():
-public protocol ExpressibleAs${node.syntax_kind} {
+%     expressible_as_type = 'ExpressibleAs' + node.syntax_kind
+%     expressible_as_protocols = SYNTAX_BUILDABLE_EXPRESSIBLE_AS_CONFORMANCES.get(expressible_as_type)
+%     if expressible_as_protocols:
+public protocol ${expressible_as_type}: ${', '.join(expressible_as_protocols)} {
+%     else:
+public protocol ${expressible_as_type} {
+%     end
   func create${node.syntax_kind}() -> ${node.syntax_kind}
 }
 
-extension ${node.syntax_kind}: ExpressibleAs${node.syntax_kind} {
+extension ${node.syntax_kind}: ${expressible_as_type} {
   public func create${node.syntax_kind}() -> ${node.syntax_kind} {
     self
   }
 }
+%     if expressible_as_protocols and 'ExpressibleAsConditionElementList' in expressible_as_protocols:
```

Comment:
It’s a shame we need to split those conforming extensions to two places… What do you think about adding all of them toward the end of `Buildable.swift.gyb` by iterating over all keys in `SYNTAX_BUILDABLE_EXPRESSIBLE_AS_CONFORMANCES`?

Reply:
Makes it more clean now.